Commands/Commands.cs

```diff
@@ -23,14 +24,21 @@ public async Task GetLatency(CommandContext ctx)
                 .Build();
             await ctx.Channel.SendMessageAsync(embed: latency);
         }
-        [Command("F")]
-        private async Task F()
-        {
-            await ReplyAsync("F");
-        }
-        [Command("say")]
-        [Summary("say what user gave as input")]
-        public Task SayAsync([Remainder] [Summary("What shoukd the bot say")] string echo)
-            => ReplyAsync(echo);
+
+        //F command
+        [Command("F"), Aliases("f")]
+        [RequiresRole("Member")]
+        [Cooldown(3, 120, CooldownBucketType.User)]
+        [Description("Returns F")]
```

Comment:
That isn't really a clear description. When making a description, you have to know that it will appear in the help command.